edge_computing/image-base-install/ibi-understanding-image-based-install.adoc

```diff
@@ -10,11 +10,15 @@ Image-based installations streamline the deployment process for {sno} clusters b
 
 This approach enables the preinstallation of configured and validated instances of {sno} on target hosts. These preinstalled hosts can be rapidly reconfigured and deployed at the far edge of the network, including in disconnected environments, with minimal intervention.
 
+[NOTE]
+====
+To deploy a managed cluster using an imaged-based approach in combination with {ztp-first}, you can use the SiteConfig Operator. For more information about the SiteConfig Operator, see link:https://docs.redhat.com/en/documentation/red_hat_advanced_cluster_management_for_kubernetes/[Red Hat Advanced Cluster Management for Kubernetes].
```

Comment:
Well, I can't find anything about 'Siteconfig Operator' - maybe we should change  to GitOps Operator

Reply:
We don't have the SiteConfig Operator docs released yet, that will be available with ACM 2.12 on 29th October. I'll update the link here before we merge this PR.